extensions/interactions/RatioExpressionInput/directives/ratio-expression-input-validation.service.spec.ts

```diff
@@ -297,6 +297,17 @@ describe('RatioExpressionInputValidationService', () => {
     }]);
   });
 
+  it('should catch integral value greater than 10 for # terms', () => {
```

Comment:
I think we should also add another edge case where the value is equal to 10. (The warning list should be empty in that case)